src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -101,6 +101,128 @@ class OPENASSETIO_CORE_EXPORT Manager final {
+   * @{
+   */
+
+  /**
+   * Capabilities that the manager implements.
+   *
+   * Many OpenAssetIO methods are optional. This enum is used with the
+   * introspection mechanism @ref hasCapability to provide a means of
+   * querying which sets of methods the manager provides.
+   */
+  enum class Capability : int {
+    /**
+     * Manager makes use of the context to persist custom state for
+     * performance reasons or otherwise.
+     *
+     * @note If this capability is true, then the host must reuse the
+     * same context across related API calls (including the use of @ref
+     * persistenceTokenForContext when the calls are distributed
+     * cross-process).
+     *
+     * @see @ref stable_resolution
+     */
+    kStatefulContexts =
+        static_cast<int>(managerApi::ManagerInterface::Capability::kStatefulContexts),
+    /**
+     * Manager customizes certain human-readable strings that the host
+     * might want to use in UI/messages.
+     *
+     * This capability means the manager implements the following
+     * methods:
+     * - @ref updateTerminology
+     */
+    kCustomTerminology =
+        static_cast<int>(managerApi::ManagerInterface::Capability::kCustomTerminology),
+    /**
+     * Manager is capable of resolving @ref entity_reference into
+     * the data for one or more @ref trait "traits."
+     *
+     * This capability means the
+     * manager implements the following methods:
+     * - @ref resolve
+     */
+    kResolution = static_cast<int>(managerApi::ManagerInterface::Capability::kResolution),
+    /**
+     * Manager allows the host to create or update an @ref entity
+     * within the @ref asset_management_system
+     *
+     * This capability means the
+     * manager implements the following methods:
+     * - @ref preflight
+     * - @ref register_
+     */
+    kPublishing = static_cast<int>(managerApi::ManagerInterface::Capability::kPublishing),
+    /**
+     * Manager is capable of querying entity references that are related
+     * to the input references by the relationship defined by a set of
+     * traits and their properties.
+     *
+     * This capability means the manager implements the following methods:
+     * - @ref getWithRelationship
+     * - @ref getWithRelationships
+     * - @ref getWithRelationshipPaged
+     * - @ref getWithRelationshipsPaged
+     */
+    kRelationshipQueries =
+        static_cast<int>(managerApi::ManagerInterface::Capability::kRelationshipQueries),
+    /**
+     * Manager is capable of confirming the existence of entities
+     *
+     * This capability means the manager implements the following
+     * methods:
+     * - @ref entityExists
+     */
+    kExistenceQueries =
+        static_cast<int>(managerApi::ManagerInterface::Capability::kExistenceQueries),
+
+    /**
+     * Manager may be capable of a providing an @ref EntityReference
+     * considered to be a sensible default for each of the given entity
+     * @ref trait_set "trait sets"
```

Comment:
[fix] Misleading to say "each of the given entity trait sets", since none are given as part of this function. Suggest just removing that bit.

Reply:
Reworded in fe470196

Reply:
lovely 👍